frontend/src/app/(editor)/@appBar/n/[noteId]/editor-app-bar.tsx

```diff
@@ -16,6 +16,21 @@ import React from 'react'
  */
 export const EditorAppBar: React.FC = () => {
   const isSynced = useApplicationState((state) => state.realtimeStatus.isSynced)
+  const noteDetailsExist = useApplicationState((state) => !!state.noteDetails)
 
-  return <BaseAppBar>{isSynced ? <NoteTitleElement /> : <RealtimeConnectionAlert />}</BaseAppBar>
+  if (!noteDetailsExist) {
+    return <BaseAppBar />
+  } else if (isSynced) {
```

Comment:
As we return in the if case above, this else is unnecessary and could be removed for easier code understanding